src/pages/home/report/ReportActionCompose.js

```diff
@@ -249,9 +258,11 @@ class ReportActionCompose extends React.Component {
      * Focus the search input in the emoji picker.
      */
     focusEmojiSearchInput() {
-        if (this.emojiSearchInput) {
-            this.emojiSearchInput.focus();
-        }
+        this.emojiFocusInteractionHandle = InteractionManager.runAfterInteractions(() => {
+            if (this.emojiSearchInput && !this.props.isSmallScreenWidth) {
```

Comment:
Why only when we have a small screen width what about tablet ?

Reply:
Yeah, this is a bigger device (!isSmallScreenWidth) but we can remove this check as `emojiSearchInput` is already null for native devices.